File: CPP_Programming/Day_4/program18.cpp
//Description : Write a program tha accepts number from the user & display its digits

#include<iostream>
using namespace std;

class Digits
{
    public :
        int iNo;

    Digits(int iNo)
    {
        this->iNo = iNo;
    }

    void Display()
    {
        int iDigit = 0;

        while(iNo != 0)
        {
            iDigit = iNo % 10;
            cout<<iDigit<<"\t";
            iNo = iNo /10;
        }
    }
};


int main()
{
    int iValue = 0;

    cout<<"Enter the number : ";
    cin>>iValue;

    Digits dobj(iValue);

    dobj.Display();

    return 0;
}

File: Java_Programming/Day_4/program18.java
//Description : Write a program tha accepts number from the user & display its digits

import java.util.*;
class Digits
{
    public int iNo;

    Digits(int iNo)
    {
        this.iNo = iNo;
    }

    public void Display()
    {
        int iDigit = 0;

       while(iNo != 0)
       {
            iDigit = iNo % 10;
            System.out.print(iDigit+"\t");
            iNo = iNo / 10;
       }
    }
}

class program18
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the number : ");
        int iValue = sobj.nextInt();

        Digits dobj = new Digits(iValue);

        dobj.Display();

        sobj.close();
    }
}